apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/SpannerConnection.cs

```diff
@@ -110,6 +110,12 @@ public override ConnectionState State
 
         internal bool IsOpen => (State & ConnectionState.Open) == ConnectionState.Open;
 
+        /// <summary>
```

Comment:
Currently this is only available as a property in the connection. Do you think we want people to be able to provide it in the connection string itself? I don't know what the expected use is likely to be.

We could potentially do that later.